### PR TITLE
Web file repository - Added a new command: wfr-upload-as-file

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Added a new command which is wfr-upload-as-file to upload a file from text data.

[https://paloaltonetworks.zoom.us/rec/share/Rgurl03svTwwa534eMfi7Oh4mwfPs4ddkeT7-Aebs-MZ-tbSCo_9vgrePeHauQYe.yjt5ijT7legqlmRo](https://urldefense.proofpoint.com/v2/url?u=https-3A__paloaltonetworks.zoom.us_rec_share_Rgurl03svTwwa534eMfi7Oh4mwfPs4ddkeT7-2DAebs-2DMZ-2DtbSCo-5F9vgrePeHauQYe.yjt5ijT7legqlmRo&d=DwMFaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=Ofm-xnwIyR-wX5Qd5Wta5yebOWzrIzmkRd55xrRodnY&m=oVUAKcn_98hN4wv18Q4Fx1HKYVD4Qnl_tgEEzg2mb8wXfK6Bx-5OOJjZDLa2wcjU&s=GNtT3_PBbyeHUkTaJgzaw65yxYXzbel55dx0CkNRpPA&e=)
Passcode: MA3^BJx3L@

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/WebFileRepository/ReleaseNotes/1_0_23.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Web File Repository
+
+- Added wfr-upload-as-file.
```
Comment (yucohen) [inline Packs/WebFileRepository/ReleaseNotes/1_0_23.md:6]:
```suggestion
- Added the ***wfr-upload-as-file*** command.
```

**Review by yucohen — CHANGES_REQUESTED** [review]
@spearmin10 Thanks for your contribution! great work! just one small change :)

File: Packs/WebFileRepository/ReleaseNotes/1_0_23.md (hunk 1 above)
Comment (spearmin10) [reply Packs/WebFileRepository/ReleaseNotes/1_0_23.md]: @yucohen I've fixed it.

**Review by yucohen — APPROVED** [review]
Great!

**Pull request merged**